<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Follow</class>
 <widget class="QDialog" name="Follow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Follow{ 
	background-image: url(:/fl.jpg);
}

#label{
color:white
;
}
#lineEdit{
background-color:rgba(255, 255, 255,1);
color:black;
}
QPushButton{
background-color:#362854;
color:black;
}

QLineEdit {
background-color:rgba(255, 255, 255,0.5);
color:white;
}

QTextEdit {
background-color:rgba(82, 82, 82,0.5);
color:white;
}
#frame_2{
	background-image: url(:/icon.png);
}
</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>680</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>20</width>
     <height>641</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FVF Fernando 08</family>
     <pointsize>18</pointsize>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <fontweight>Black</fontweight>
    </font>
   </property>
   <property name="text">
    <string>Instagram</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>760</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FVF Fernando 08</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>FOLLOWER</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>830</y>
     <width>311</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FVF Fernando 08</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Information</string>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>20</width>
     <height>641</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>371</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FVF Fernando 08</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FVF Fernando 08</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="placeholderText">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QPushButton" name="followButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>580</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FVF Fernando 08</family>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Follow</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
